GPRegistryPolicyParser.psm1

```diff
@@ -449,6 +390,8 @@ Function Create-RegistrySettingsEntry
         [GPRegistryPolicy]
         $RegistryPolicy
     )
+
+    Write-Host "Creating key $($RP.KeyName)"
```

Comment:
Please try not to use Write-Host inside scripts http://www.jsnover.com/blog/2013/12/07/write-host-considered-harmful/